config/output.go

```diff
@@ -36,7 +36,7 @@ func (t *Config) RunOutputs() (err error) {
 func (t *Config) runOutputs(evchan chan logevent.LogEvent, logger *logrus.Logger) (err error) {
 	outputs, err := t.getOutputs()
 	if err != nil {
-		return errutil.New("get outputs failed", err)
+		return err
```

Comment:
I think it's better to wrap the output module error, because the output module may not throw enough error information.